.sync.yml

```
@@ -1,3 +1,3 @@
 ---
 spec/spec_helper.rb:
-  hiera_config: spec/fixtures/hiera.yaml
+  hiera_config: "'spec/fixtures/hiera.yaml'"
```

Now that we know the override isn't needed, I think `spec_helper_acceptance.rb` is completely in line so we can manage it:
```suggestion
  hiera_config: "'spec/fixtures/hiera.yaml'"
spec/spec_helper_acceptance.rb:
  unmanaged: false
```

---

Hmm, strange double negative there :)